string cmd
float32 position0
float32 position1
float32 position3
string gripper_cmd
